pkg/controllers/chaperon/controller.go

```diff
@@ -113,13 +114,23 @@ func (c *reconciler) Handle(obj interface{}) (requeueAfter *time.Duration, err e
 		if !errors.IsNotFound(err) {
 			return nil, err
 		}
-
-		podNeverExisted := podChaperon.Status.Phase == ""
+		podNeverExisted := podNeverExisted(podChaperon)
 		if podNeverExisted || podMissing && time.Since(podMissingSince) > PodRecreatedIfPodChaperonNotDeletedAfter && podChaperon.Spec.RestartPolicy == corev1.RestartPolicyAlways {
 			var err error
 			pod, err = c.kubeclientset.CoreV1().Pods(podChaperon.Namespace).Create(ctx, newPod(podChaperon), metav1.CreateOptions{})
 			if err != nil {
-				return nil, fmt.Errorf("cannot create pod for pod chaperon %v", err)
+				// conditions will be updated with pod conditions upon successful creation
+				podChaperon.Status.Conditions = []corev1.PodCondition{
+					{
+						Type:               corev1.PodScheduled,
+						Status:             corev1.ConditionFalse,
+						Reason:             common.PodReasonFailedCreate,
+						LastTransitionTime: metav1.Now(),
+						Message:            err.Error(),
+					},
+				}
+				podChaperon, _ = c.customclientset.MulticlusterV1alpha1().PodChaperons(podChaperon.Namespace).UpdateStatus(ctx, podChaperon, metav1.UpdateOptions{})
```

Comment:
ignoring handling the error here, because we primarily care about the first error from the failed creation and eventually, you'd hope the status update would succeed if lets say conflicts occur on the first update, etc.